packages/expo-file-system/ios/Next/FileSystemNextModule.swift

```diff
@@ -10,7 +10,8 @@ public final class FileSystemNextModule: Module {
       return [
         "documentDirectory": appContext?.config.documentDirectory?.absoluteString,
         "cacheDirectory": appContext?.config.cacheDirectory?.absoluteString,
-        "bundleDirectory": Bundle.main.bundlePath
+        "bundleDirectory": Bundle.main.bundlePath,
+        "appleSharedContainers": getAppleSharedContainers()
```

Comment:
comment: we should probably move this to properties at some point so it's lazy eval, but this requires some core changes